### PR TITLE
fix: CineDialog buttons label

### LINKED ISSUE #995: CINE control buttons with wrong titles
### Description

To improve user experience, we would like to have labels/titles coherent with its button's functionality.

On CINE control, the current labels/titles of buttons 'Next image' and 'Skip to last image' may mislead the user, since they are not coherent with the functionality of each button.

cine-labels

### Steps to Reproduce

1. Load view studies page
2. Select the Cine tool on the toolbar
3. Hover on the 4th button and verify that the label says "Play Image"
4. Hover on the 5th button and verify that the label says "Skip, to last image"

### Acceptance Criteria

To be coherent with its functionality:
-  the 4th button should have title/label "Next image"
-  the 5th button should have title/label "Skip to last image"

### PR DESCRIPTION
**Pull request by rodrigolabs**
Fixes #995

### CODE REVIEW

File: platform/i18n/src/locales/en-US/CineDialog.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "Next image": "$t(Common:Play) $t(Common:Image)",
+  "Next image": "$t(Common:Next) $t(Common:Image)",
   "Play / Stop": "$t(Common:Play) / $t(Common:Stop)",
   "Previous image": "$t(Common:Previous) $t(Common:Image)",
   "Skip to first image": "Skip to first $t(Common:Image)",
-  "Skip to last image": "Skip, to last $t(Common:Image)",
```
Comment (swederik) [inline platform/i18n/src/locales/en-US/CineDialog.json]: Do not change this here, we need to change it in Locize. You can remove this change from the PR: https://github.com/OHIF/Viewers/blob/48ea7a28e50b845c4939a8aaf2d95eba49d2c102/platform/i18n/src/index.js#L11

**Comment by dannyrb** [comment]
## Steps to update translation mappings:

image

image

- Click the `OHIF` project
- Find the target language `en-US` (our reference language)
    - Click "CineDialog" under our language
    - Update values/mappings
    - Click Save
- Pull updated language files using `Locize` API

See:
`Viewers/platform/i18n/pullTranslations.sh`: [Here](https://github.com/OHIF/Viewers/blob/da725a892382cfed58c534e0b76872a961185006/platform/i18n/pullTranslations.sh)

The `.locize` configuration file is not checked into source code (so we don't leak our API key). Here is the shape of the configuration file:

- https://github.com/locize/locize-cli#other-information

I'm sending you an invitation to `locize` so you can access our Project's API key and do these edits. I think only @swederik has done this end-to-end so far. Let me know if you encounter issues and I can assist.

**Comment by rodrigolabs** [comment]
@swederik @dannyrb Now, all changes were pulled from Locize

**Pull request merged**